### DISCUSSION TITLE
changed timezone and now not booting

### DISCUSSION THREAD

**Discussion by Marcoske23** [gtxaspec/wz_mini_hacks#543] [Q&A]
I have problems with the camera schedule so I was looking for a way to fix it.
Since the Wyze app didn't fix it, I opted to directly modify the ".wz_backup\configs\.user_config" with a Windows PC.
When the camera was restarted, only the red light was on.
I tried changing the firmware, factory reset, starting the wz_mini_hack from scratch.
None of the above options worked.
Could someone guide me on how to solve the problem or the camera is brick?
Thanks!

**Comment by endertable** [comment]
You may have "corrupted" the .user_config file to where the camera couldn't read it.  You should not use Windows PC for editing Linux config files, unless you know how to save the file correctly.  Edit using vi from the camera's shell if you can.

You should not have to have redone all that to fix your issue, unless it was something else, altogether.  Based on what you did, now there are other questions to ask:

What FW did you change to?

What  do you mean by you "factory reset"?

How did you start the wz_mini from scratch, do you mean you reformatted the SD card, or just removed and reinstalled the wz_mini directory?

I would say your camera should not be bricked, but we'll see...

**Reply by Bob-AOL** [reply]
If Notepad is used to edit a Unix (LF) text file the formatting will generally remain Unix(LF) and is easily verified by the format being displayed at the bottom of the editing window. 

![Notepad with Unix(LF)](https://github.com/gtxaspec/wz_mini_hacks/assets/110629141/8f13e642-6d6d-40be-9f4b-1b1abf0aa438)

**Reply by Marcoske23** [reply]
What FW did you change to?
from 4.36.9.139 to 4.61

What do you mean by you "factory reset"?
press setup button 10 seconds.

How did you start the wz_mini from scratch, do you mean you reformatted the SD card, or just removed and reinstalled the wz_mini directory?
I formatted the sd card and installed new wz_mini directory.

I created a code that allows me to return the .wz_configs I had backed up to the /configs/ folder. The camera worked the first time but after adding the new version of the wz_mini from scratch it did not turn the camera back on. I had a new camera unused and installed new wz_mini version only modifying to ethernet, he did not turn on the camera either.

**Reply by Bob-AOL** [reply]
The camera needs to be up and running in wifi mode to enable ethernet and will not boot otherwise.

**Reply by Marcoske23** [reply]
I know that, first I configure the camera with the app and then I put the wz_mini. Once inside connected by wifi I usually use the webserver to make modifications and at that time I connect the ethernet and then restart. Anyway, the camera should work if I remove the micro sd, which does not happen.

**Reply by Bob-AOL** [reply]
I would try another SD card.

**Reply by endertable** [reply]
@Marcoske23 you are right, with no SD card, you should be able to still configure the cam from a fresh factory reset.  That would be a good starting point to determine if the cam has not been bricked.  I am sure you found these, but just in case:

https:  //support.wyze.com/hc/en-us/articles/360055968891-How-to-reset-your-Wyze-Cam-v3

**Reply by Bob-AOL** [reply]
Won't functioning firmware be required for a factory reset?

**Comment by gtxaspec** [comment]
see https://github.com/gtxaspec/wz_mini_hacks/issues/329
